delfin/api/schemas/access_info.py

```diff
@@ -39,12 +39,11 @@
                              'maxLength': 255},
                 'password': {'type': 'string', 'minLength': 1,
                              'maxLength': 255},
-                'host_key': {'type': 'string', 'minLength': 1,
-                             'maxLength': 255},
-                'host_key_type': parameter_types.host_key_type
+                'pub_key': {'type': 'string', 'minLength': 1,
+                            'maxLength': 255},
+                'pub_key_type': parameter_types.host_key_type
             },
-            'required': ['host', 'port', 'username', 'password', 'host_key',
-                         'host_key_type'],
+            'required': ['host', 'port', 'username', 'password', 'pub_key'],
             'additionalProperties': False
```

Comment:
pub_key is optional, plz update openapi.yaml also

Reply:
We can update openapi.yml as below.. 
![Screenshot from 2020-08-24 14-39-39](https://user-images.githubusercontent.com/45681499/91026530-7d0f9c00-e618-11ea-8ceb-a64f81d13a6a.png)
jusr remove pub_key and pb_key_type from required